src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,44 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int STACK_MAX_SIZE = 12;
+    private static final int NEGATIVE_NUMBER = -1;
+    private static final int FULL_INDEX = 11;
```

Comment:
This constant is related to STACK_MAX_SIZE . Changing only one leads to an error (inconsistency).